docs/sdkx-react-native/users.mdx

```diff
@@ -0,0 +1,299 @@
+
+#### Sending the HMAC Digest to verify users' identity
+
+You can send the `user authentication token` with the `login` API call. If Identity Verification is enabled on the Helpshift Dashboard (documentation [here](https://support.helpshift.com/kb/article/what-is-user-identity-verification-and-how-do-i-set-it-up/)), Helpshift recalculates the unique 'user authentication token' using the shared secret key and compares the 'user authentication token' sent by you to the recalculated value. If they're equal, then the user's identity is verified, and they can file Issues.
+
+You can use the `userAuthToken` key to pass into the `login` API map.
+
+Example :
+
+```javascript
+import {
+  login
+} from 'helpshift-plugin-sdkx-react-native';
+
+let config = {
+  userName: 'John Doe',
+  userEmail: 'john.doe@app.co',
+  userId: 'unique-user-id-746501',
+  userAuthToken: 'generated-user-authentication-token'
+}
+
+login(config);
+```
+
+<Admonition type="info" title="Note">
+
+- If the secret key is regenerated for an app on the Dashboard, you should ensure you update your endpoint code as well, to generate a valid 'user authentication token'. Following this, your Admins should delete the old secret key from the Dashboard to ensure requests using a 'user authentication token' generated using the old secret key become invalid.
+
+- If EnableFullPrivacy is 'On', then the 'user authentication token' should be generated only on the user ID.
+
+</Admonition>
+
+### Identity verificaiton failure {#identity-verification-failure}
+
+#### Overview
+
+When Identity Verification is turned 'On' and a login request is made with no 'user authentication token' or with an invalid 'user authentication token', identity verification will fail. If identity verification fails, the following will hold true:
+
+- Anonymous users (users for whom the developer is not providing any identifiers) are always able to file Issues.
+- Logged-in users (users for whom the developer is providing an identifier such as user ID or email) will not be able to file Issues or see conversations if they are unverified. Unverified logged-in users will see an error on the conversation UI.
+
+If the logged-in users are verified (i.e. a valid 'user authentication token' is provided), they can see the previous issue or create a new issue. The UI for verified, logged-in users looks & works the same as it would if 'Identity verification' were turned off. Only unverified users get impacted as listed above.
+
+#### How to use the failure delegates
+
+If identity verification fails, the SDK will invoke the `AuthenticationFailedForUser(...)` delegate to notify the application of the failure:
+We defined these values in the Helpshift SDKX. 
+
+<table>
+  <tr>
+    <th>Authentication failure reason</th>
+    <th>When is it called</th>
+    <th>When to use</th>
+  </tr>
+  <tr>
+    <td>
+      HelpshiftAuthenticationFailureReason <br />
```

Comment:
do we have this class /constant in react native sdk?
"HelpshiftAuthenticationFailureReason"

Reply:
Updated